web/admin/src/Question.js

```diff
@@ -55,6 +55,13 @@ export default class Question extends PureComponent {
     ) {
       this.startTimer(nextProps)
     }
+    if (nextProps.startCountdown) {
```

Comment:
These indents seem off

Reply:
Do users of `<Question />` ever actually pass a `startCountdown` prop?

Reply:
Yes startCountdown is passed from PresentationDriver to start countdown only when "start question" is pressed